app/main.py

```diff
@@ -1,7 +1,27 @@
+import requests
+
+
+API_KEY = "967cf4e71dfe436ba53110325230808"
```

Comment:
also did u read a task ? a user that pulled an image should add his own api_key through env vars )